css/master.css

```diff
@@ -334,3 +334,93 @@
   padding-top: 30%;
   text-align: center;
 }
+
+.contact {
+  margin-top: var(--section-main-padding);
+  margin-bottom: var(--section-main-padding);
+}
+
+.contact .contact-header {
+  font-size: 32px;
+  font-weight: bold;
+  text-align: center;
+  color: #172b4d;
+  margin-bottom: 60px;
+}
+
+.contact form .text-email input {
```

Comment:
- [ ] It's one of the requirements to use a CSS Flexbox property to layout the form elements, please add it.